app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,41 @@
-<h2>Log in</h2>
+<div class="container-fluid p-0 m-0">
+    <div class="image-container">
+        <div class="wrap-image">
+            <div class="form-container">
+                <%= image_tag 'login-img.png', class: "mx-auto d-block img-fluid" %>
+                <hr>
+                  <% flash.each do |type, msg| %>
+                  <div class="alert alert-info">
```

Comment:
type is usually used to change alert color, something like `class="alert alert-#{type}"` and customize background color per message type

Reply:
Sweet!

Reply:
We implemented a helper function to change the background of alerts, messages look better